src/main/java/de/dhbw/mh/slang/craft/parser/CraftedSlangParser.java

```diff
@@ -232,32 +234,56 @@ AstNode relations5( AstNode previous ){
 	
 	@Override
 	public AstNode additiveExpression( ){
-		// TODO Auto-generated method stub
-		return super.additiveExpression( );
+		if( Selector.SUMMAND.contains(LEXER.lookahead().TYPE) ){
+			AstNode previous = multiplicativeExpression();
+			return summand( previous );
+		}
+		throw parsingException( Selector.SUMMAND );
 	}
 	
 	@Override
 	public AstNode summand( AstNode previous ){
-		// TODO Auto-generated method stub
-		return super.summand( previous );
+		Token token = LEXER.lookahead();
+		if( Selector.SUMMAND3.contains(token.TYPE) ){
+			return summand3( previous );
+		}
+		if( PLUS == token.TYPE ){
+			return summand1( previous );
+		}
+		if( MINUS == token.TYPE ){
+			return summand2( previous );
+		}
+		throw parsingException( Selector.SUMMAND );
 	}
 	
 	@Override
 	AstNode summand1( AstNode previous ){
-		// TODO Auto-generated method stub
-		return super.summand1( previous );
+		if (PLUS == LEXER.lookahead().TYPE) {
+			CodeLocation location = LEXER.lookahead().BEGIN;
+			LEXER.advance();
```

Comment:
Instead of the `if` statement and the advance method you could also use match(Token.Type.PLUS). But that's fine, it's basically the same thing.